stable/rabbitmq/templates/statefulset.yaml

```diff
@@ -0,0 +1,150 @@
+apiVersion: apps/v1
```

Comment:
Works only with Kubernetes 1.9+, before this was a beta API :sob:

Reply:
see #4908

Reply:
#4909 should fix it